### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.8 es 4.39 (6 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#315] [Esercizi M-Z]
![es39](https://github.com/user-attachments/assets/9e75d9f9-ba9e-41d9-9f5e-748c333e52f7)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/8e3a6c65-4d21-4996-8e68-d8368fba27bb)
